.github/workflows/integrate.yml

```diff
@@ -0,0 +1,16 @@
+name: Tests
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test_pull_request:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
```

Comment:
@Saad-Amjad 
We don't have any npm script called `ci` in our `package.json`.
So will this step run successfully? :thinking:

Reply:
npm ci is now a default command that ensures a clean install of the dependencies. So it will run if I am not mistaken. It ran here: https://github.com/monstar-lab-oss/nestjs-starter-rest-api/runs/963333776?check_suite_focus=true

Reply:
Got it.
Thanks for explaining it. :bow: